### DISCUSSION TITLE
Set first enabled day to tabindex="0"

### DISCUSSION THREAD

**Discussion by timarney** [gpbl/react-day-picker#1115] [Ideas]
Wondering if you would accept a PR to alter the code below for Day to accept a prop to select a specific date vs defaulting to the first day of the month (tabIndex 0).

```js
let tabIndex = -1;
    // Focus on the first day of the month
    if (this.props.onDayClick && !isOutside && day.getDate() === 1 
      // default to 1 but allow a prop to override
) {
      tabIndex = this.props.tabIndex; // eslint-disable-line prefer-destructuring
}
```

In most cases the first day makes sense but, I have a use case where a month can have many disabled days.  Users navigating with a screen reader end up navigating a lot of disabled days before finding an enabled date.  In which case I would like to be able to set the first enabled day to tabIndex 0.

**Comment by duff-tock** [comment]
I second this change. the first enabled day or the currently selected day should be tabIndex 0 not necessarily the first of the month.

**Comment by cjwiseman11** [comment]
Is there currently a work around for this? I'm looking to hide disabled days (visibility: hidden) but this breaks keyboard functionality as the tabindex is always -1. Is there a way to set the tabindex for each day?
